ansible/idr-haproxy.yml

```diff
@@ -0,0 +1,62 @@
+# Setup the IDR haproxy for non-web requests
+
+# Load hostvars (production OMERO)
+- hosts: >
+    {{ idr_environment | default('idr') }}-omeroreadonly-hosts
+
+
+- hosts: "{{ idr_environment | default('idr') }}-proxy-hosts"
+
+  pre_tasks:
+
+  - name: Get omero IP
+    set_fact:
+      omero_omeroreadonly_hosts: >-
+        {{
+          groups[idr_environment | default('idr') + '-omeroreadonly-hosts'] |
+          map('extract', hostvars,
+            ['ansible_' + (idr_net_iface | default('eth0')), 'ipv4', 'address']) | sort
+        }}
+
+  # https://stackoverflow.com/a/35608380
+  # TODO: Find a nicer way to do this instead of a set_fact loop
```

Comment:
The majority of the TODOs are already know by us, and related to the incomplete read-only work. I don't think more cards are needed.